### PR TITLE
CDP - Remove SQL relationships

### PR DESCRIPTION
**Pull request by LucGenetier**
#2752

### CODE REVIEW

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/libraries/Microsoft.PowerFx.Connectors/Tabular/CdpTableResolver.cs [hunk 1]
```diff
@@ -104,47 +107,51 @@ public async Task<ConnectorType> ResolveTableAsync(string tableName, Cancellatio
...
-                    ParentTable = fk.Parent_Table,
-                    ColumnName = fk.Parent_Column,
-                    ReferencedTable = fk.Referenced_Table,
-                    ReferencedColumnName = fk.Referenced_Column
-                });
-            }
-
-            return sqlRelationShips;
-        }
+        // Only keeping code for reference
```
Comment (MikeStall) [inline src/libraries/Microsoft.PowerFx.Connectors/Tabular/CdpTableResolver.cs:110]: nit - consider `#if false`?

**Review by MikeStall — APPROVED** [review]
:shipit:

File: src/libraries/Microsoft.PowerFx.Connectors/Tabular/CdpTableResolver.cs (hunk 1 above)
Comment (LucGenetier) [reply src/libraries/Microsoft.PowerFx.Connectors/Tabular/CdpTableResolver.cs:110]: Just a different syntax, yes

**Comment by LucGenetier** [comment]
✅ No public API change.

**Pull request merged**